assets/js/util.js

```diff
@@ -119,6 +119,25 @@ $(document).ready(function () {
     $('.modal').on('hide.bs.modal', function () {
         $('a[data-target=#' + $(this).attr('id') + ']').parents('li').removeClass('active');
     });
+
+    $(window).scroll(function () {
+        if($(window).scrollTop() > BACK_TO_TOP_ACTIVATION_HEIGHT) {
+            $('#backToTop').removeClass('hidden-xs hidden-sm hidden-md hidden-lg');
+            $('#backToTop').addClass('hidden-xs hidden-sm visible-md visible-lg');
```

Comment:
Um, yep these do make sense! I have modified the PR and added `hidden-xs hidden-sm` to the HTML element directly. I only fiddle around with the `visible/hidden-md,lg` in the JS.
Thanks @sushain97  for this! It definitely makes a lot more sense to me now.